aiida/tools/importexport/common/archive.py

```diff
@@ -304,36 +61,55 @@ def extract_zip(infile, folder, nodes_export_subfolder=None, **kwargs):
         incorrect formats
     """
     # pylint: disable=fixme
+
     if nodes_export_subfolder:
         if not isinstance(nodes_export_subfolder, str):
             raise TypeError('nodes_export_subfolder must be a string')
     else:
         nodes_export_subfolder = NODES_EXPORT_SUBFOLDER
 
+    if not kwargs.get('silent', False):
```

Comment:
In the export, the progress is now being set by the CLI command.
Is there a reason to take a different approach here?

Reply:
this was simply for back compatibility. these functions are no longer actually used for read/write, only in the archive migrations where the CLI has not yet been changed.
When I refactor that (I think now in a separate PR) I will probably remove this

Reply:
I'll keep this open in case you ever look back to this PR